packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -60,11 +170,24 @@ export function OpLatencyView(): React.ReactElement {
 				}}
 				yAxisUnitDisplayName="ms"
 				// NOTE: Because Op Latency data is not yet available, this is a placeholder
-				dataSets={[]}
+				dataSets={[
+					durationOutboundBatchingData,
+					durationNetworkData,
+					durationInboundToProcessingData,
+				]}
 			/>
 			<div className={styles.graphAboutContainer}>
 				<div className={styles.flexColumn}>
 					<Subtitle1>About</Subtitle1>
+					{renderText && (
```

Comment:
Let's make the flag render _either_ the instructions or the other content. If we always render the chart, I'm sure some users won't pay attention to the text above it so they might not realize things are not configured quite right. And I think `renderInstructions` as name might be a bit more explanatory.